src/lint/mod.rs

```diff
@@ -125,7 +126,7 @@ impl LinterAnalysis {
         let local_lint_errors = begin_style_check(original_analysis.ast, file.text, &rules)?;
         let mut lint_errors = vec![];
         for error in local_lint_errors {
-            lint_errors.push(error.warning_with_file(path));
+            lint_errors.push(error.error.warning_with_file(path));
```

Comment:
Rename error to entry in this for loop, so it reads `entry.error.warning_with_file(path)` when doing the push